sdcm/sct_config.py

```diff
@@ -1775,6 +1785,18 @@ def __init__(self):
                 raise ValueError("For PasswordAuthenticator authenticator authenticator_user and authenticator_password"
                                  " have to be provided")
 
+        # 13) validate stress and prepare duration:
+        if stress_duration := self.get('stress_duration'):
```

Comment:
Why the `prepare_duration` depends on the setting of the `stress_duration`?

Reply:
The stress duration set  duration of stress commands. But before that we run prepare commands, which build the dataset.
if stress_duration doestn' set (old behavior), the test_duration already include time which is need to run prepare dataset commands.
But if we set the stress duration, then new test duration should be counted based on provided stress duration and time for building dataset. But different job use different dataset which is generated for different time. One of advantage is that if prepare stress cmd finished ealier, then we don't wait full time. So after discussion we decide to use most longest time for prepare command is ~5h. in future we may want to configure prepare time. and based on it we should count test_duration.

So for new behavior, if stress_duration is set we additionally have to get time for prepare stress commands if it is not set, then default will be used, and new test duration will be calculated.

Reply:
I understood the logic of the implementation. I just disagree that it is correct way for doing it.
The `stress prepare` and `main stress` commands run only in separate time frames and both combine the main test run spent time excluding the provisioning part.
So, from my point of view, both are equal and it should be possible to handle them separately.

Reply:
I agree with @vponomaryov those two should work independently

Reply:
fixed